Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+library("tdr-jenkinslib")
+
+pipeline {
+  agent {
+    label "master"
+  }
+
+  stages {
+    stage("Run git secrets") {
+      steps {
+        script {
+          def exitCode = sh(script: "git-secrets --scan", returnStatus: true)
+          def scanOutput = sh(script: "git-secrets --scan 2>&1 >/dev/null | head -n 4 | cut -c1-6 | tr -d '\n'", returnStdout: true)
+          if (exitCode == 0 || scanOutput.trim() != "test:1test:3test:5") {
+            tdr.postToDaTdrSlackChannel(colour: "danger", message: "*Security test failure*: expected Jenkins to detect sensitive information but it did not. Check that git-secrets is installed and configured on Jenkins.\nhttps://jenkins.tdr-management.nationalarchives.gov.uk/job/Git%20Secrets%20Check/")
```

Comment:
You can add a link to the actual build with the environment variable `BUILD_URL`.

Reply:
Good point. I've changed it.

Reply:
Have you just updated the one below? I think it makes sense to use the build URL here too.

Reply:
Ah yeah, sorry. I've updated the other message as well.